config/initializers/setup_mail.rb

```diff
@@ -13,11 +13,13 @@
   ActionMailer::Base.default_url_options[:host] = ENV['DOMAIN']
 end
 
+
 unless ENV['EMAIL_OVERRIDE'].nil?
     class OverrideMailRecipient
         def self.delivering_email(mail)
             mail.body = "DEVELOPMENT-OVERRIDE. Was being sent to " + mail.to.first + "\n" + mail.body.to_s
             mail.to = ENV['EMAIL_OVERRIDE']
+            mail.bcc = ENV['EMAIL_OVERRIDE']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.